test/tla/TwoPhaseTyped_typedefs.tla

```diff
@@ -0,0 +1,7 @@
+----------------------- MODULE TwoPhaseTyped_typedefs ----------------
+(*
+ @typeAlias: MESSAGE = Commit(NIL) | Abort(NIL) | Prepared(RM);
```

Comment:
This makes me wonder, should we reserve a `Unit`-like type for stuff like this, so it isn't ad-hoc implemented?

Reply:
Yes, but what I'm saying is we should create a list of reserved words (not just for variants, globally), and `Unit` or `NIL` or something alike should be on it. You can't expect people to figure out this idiom from a GH issue.

Reply:
Followed by static analysis for reserved words.

Reply:
We will write about it in the howto and the tutorial.